embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/arch/SessionSpanWriter.kt

```diff
@@ -28,8 +28,8 @@ internal interface SessionSpanWriter {
  * Represents a span event that can be added to the current session span.
  */
 internal class SpanEventData(
```

Comment:
I got confused about what this is since `SpanEvent` is already a thing.  What you want to capture here are the parameters to startSpan()? If so, it should probably include `parent` in case this data source wants to create child spans as well as parent spans?